.pre-commit-config.yaml

```diff
@@ -65,13 +65,15 @@ repos:
         language: script
         files: (pkg/graph/schema.graphql|pkg/graph/schema.resolvers.go|src/queries/*.ts)
 
+  # for some reason, trying to call "go run cmd/intake_schema_gen/main.go" directly from this config doesn't work;
+  # so call scripts/gen_intake_schema, which calls that command and works 
   - repo: local
     hooks:
-      - id: check-cedar-schema-generation
-        name: check-cedar-schema-generation
+      - id: check-intake-schema-generation
+        name: check-intake-schema-generation
         entry: scripts/gen_intake_schema
```

Comment:
I tried with this config:
```
hooks:
      - id: check-intake-schema-generation
        name: check-intake-schema-generation
        entry: gen_intake_schema
        language: golang
        files: cmd/gen_intake_schema/main\.go|pkg/cedar/intake/models/.*\.go|pkg/cedar/intake/schemas/.*\.json
```
but it doesn't seem to re-run the Go script on changes to files.

Reply:
Wouldn't you also want to change the `entry` line too?

Reply:
On further testing, this might work, I'm just having trouble testing in my local environment.

Reply:
I tried this in https://github.com/CMSgov/easi-app/pull/1450/commits/1822f064109cf0233efdc29ed695c9aa4e4b9dd7. The [pre-commit docs](https://pre-commit.com/#golang) say

> The hook repository must contain go source code. It will be installed via go install ./.... pre-commit will create an isolated GOPATH for each hook and the [entry](https://pre-commit.com/#hooks-entry) should match an executable which will get installed into the GOPATH's bin directory.

Running `go install ./...` locally puts an executable called `gen_intake_schema` in my local `GOPATH/bin` folder, but from the [CI logs](https://github.com/CMSgov/easi-app/runs/5073445180?check_suite_focus=true), it doesn't seem to get set up correctly.

I'm going to stick with the indirection of going through a script, this isn't worth spending more time on it.